rstudio/c9s-python-3.11/Dockerfile

```diff
@@ -60,6 +60,9 @@ RUN R -e "require('remotes'); \
     remotes::install_version('vetiver','0.2.5'); \
     remotes::install_version('devtools','2.4.5');"
 
+# install necessary OS package to make Knit R markup to PDF rendering possible
+RUN yum install -y texlive-framed
```

Comment:
Thank you Sven for this enhancement!
May i ask you to move this package to this [line](https://github.com/opendatahub-io/notebooks/blob/ca6889b6b49f0c67e2d467e58b1bd1499f481275/rstudio/c9s-python-3.11/Dockerfile#L51)?

Reply:
with dnf?

Reply:
moved to dnf line, please review